src/styles/main.scss

```diff
@@ -0,0 +1,63 @@
+* {
+  margin: 0;
```

Comment:
don't remove margins and paddings for all the selectors. Use reset.scss for that